locations/spiders/mavis_us.py

```diff
@@ -0,0 +1,23 @@
+from scrapy.linkextractors import LinkExtractor
+from scrapy.spiders import CrawlSpider, Rule
+
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class MavisUSSpider(CrawlSpider, StructuredDataSpider):
+    name = "mavis_us"
+    item_attributes = {"brand_wikidata": "Q65058420"}
+    start_urls = ["https://www.mavis.com/locations/all-stores/"]
+    rules = [
+        Rule(
+            LinkExtractor(allow=r"/locations/[^/]+/$", restrict_xpaths=['//a[starts-with(text(), "Mavis")]']),
```

Comment:
"Mavis" matches:
* Mavis Tire & Brakes
* Mavis Tires & Brakes
* Mavis Discount Tire

I think the first one (missing s after Tire) is just misspelled and the real brand according to the logo is "Mavis Tires & Brakes". The "Mavis Discount Tire" appears to be a different brand name with a dedicated logo.

My suggestion is that "Mavis" be changed to "Mavis Tire" for this spider, and a new spider implemented as a copy+paste of this one for "Mavis Discount Tire" and all the other listed brands like Action Gator Tire, etc.